.gitignore

```diff
@@ -1,8 +1,9 @@
+.idea
+
 # do not track dependencies
 node_modules
 
 # ignore generated code coverage output
-coverage
```

Comment:
Why removing this? It's still generated, right?

Reply:
Nope